codegen_glibmm/templates/proxy.h.templ

```diff
@@ -94,21 +96,25 @@ public:
     void reference() const override {}
     void unreference() const override {}
 
-    void handle_signal(const Glib::ustring &sender_name,
-                       const Glib::ustring &signal_name,
-                       const Glib::VariantContainerBase &parameters);
-
 protected:
     Glib::RefPtr<Gio::DBus::Proxy> m_proxy;
 
 private:
     {{ interface.cpp_class_name }}(Glib::RefPtr<Gio::DBus::Proxy> proxy): Glib::ObjectBase() {
         this->m_proxy = proxy;
         this->m_proxy->signal_signal().connect(sigc::mem_fun(this, &{{ interface.cpp_class_name }}::handle_signal));
+        this->m_proxy->signal_properties_changed().connect(sigc::mem_fun(this, &{{ interface.cpp_class_name }}::handle_properties_changed));
```

Comment:
Wrap this line after the first `.`

Reply:
I would be happy to see this constructor moved to the cpp file; you don't have to do it, but, if you have some spare time, that would be great (but if you do it, do it in a separate commit) :-)

Reply:
Done and done. (Thought about moving constructor before sending pr but held off due to conflict with #60.)